{"failedTests":[{"testName":"parallel/test-runner-watch-mode","timestamp":1725262864654,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 0,win10-COMPILED_BY-vs2022 #29840","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/29840/","buildNumber":29840,"builtOn":"test-azure_msft-win10_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"09dd58319e84d26d6053f1ba5a4a739257c767fa","callStack":[{"upstreamBuild":65401,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73805,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61811,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54552"}],"tap":{"ok":false,"name":"parallel/test-runner-watch-mode","id":713,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ test runner watch mode\n  ✖ should run tests repeatedly (6387.6265ms)\n    AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n    \n    '✔ test has ran (18.1095ms)\\n' +\n      '✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (1058.8466ms)\\n' +\n      \"  'test failed'\\n\" +\n      '\\n' +\n      '✔ test has ran (23.9442ms)\\n' +\n      '✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (957.6018ms)\\n' +\n      \"  'test failed'\\n\" +\n      '\\n' +\n      '✔ test has ran (17.7252ms)\\n' +\n      'ℹ tests 5\\n' +\n      'ℹ suites 0\\n' +\n      'ℹ pass 3\\n' +\n      'ℹ fail 2\\n' +\n      'ℹ cancelled 0\\n' +\n      'ℹ skipped 0\\n' +\n      'ℹ todo 0\\n' +\n      'ℹ duration_ms 1181.2087\\n'\n    \n        at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:80:14)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:161:26)\n        at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:170:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 0)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: true,\n      code: 'ERR_ASSERTION',\n      actual: \"✔ test has ran (18.1095ms)\\n✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (1058.8466ms)\\n  'test failed'\\n\\n✔ test has ran (23.9442ms)\\n✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (957.6018ms)\\n  'test failed'\\n\\n✔ test has ran (17.7252ms)\\nℹ tests 5\\n...\",\n      expected: /tests 1/,\n      operator: 'match'\n    }\n\n  ✔ should run tests with dependency repeatedly (4208.3525ms)\n  ✔ should run tests with ESM dependency (4046.7746ms)\n  ✖ should support running tests without a file (6053.9114ms)\n    AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n    \n    '✖ test.js (1031.0906ms)\\n' +\n      \"  'test failed'\\n\" +\n      '\\n' +\n      '✔ test has ran (10.3761ms)\\n' +\n      '✖ test.js (1016.3231ms)\\n' +\n      \"  'test failed'\\n\" +\n      '\\n' +\n      '✔ test has ran (27.7831ms)\\n' +\n      'ℹ tests 4\\n' +\n      'ℹ suites 0\\n' +\n      'ℹ pass 2\\n' +\n      'ℹ fail 2\\n' +\n      'ℹ cancelled 0\\n' +\n      'ℹ skipped 0\\n' +\n      'ℹ todo 0\\n' +\n      'ℹ duration_ms 1112.9031\\n'\n    \n        at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:80:14)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:161:26)\n        at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:182:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n      generatedMessage: true,\n      code: 'ERR_ASSERTION',\n      actual: \"✖ test.js (1031.0906ms)\\n  'test failed'\\n\\n✔ test has ran (10.3761ms)\\n✖ test.js (1016.3231ms)\\n  'test failed'\\n\\n✔ test has ran (27.7831ms)\\nℹ tests 4\\nℹ suites 0\\n...\",\n      expected: /tests 1/,\n      operator: 'match'\n    }\n\n  ✖ should support a watched test file rename (4010.2208ms)\n    Error: ENOENT: no such file or directory, rename 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test.js' -> 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test-renamed-test.js'\n        at renameSync (node:fs:1020:11)\n        at Timeout._onTimeout (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:93:40)\n        at listOnTimeout (node:internal/timers:614:17)\n        at process.processTimers (node:internal/timers:549:7) {\n      errno: -4058,\n      code: 'ENOENT',\n      syscall: 'rename',\n      path: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js',\n      dest: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test-renamed-test.js'\n    }\n\n  ✖ should not throw when delete a watched test file (8.1859ms)\n    Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n        at Object.rmSync (node:fs:1246:18)\n        at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n        at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n        at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n        at TestHook.runInAsyncScope (node:async_hooks:211:14)\n        at TestHook.run (node:internal/test_runner/test:887:25)\n        at TestHook.run (node:internal/test_runner/test:1157:18)\n        at node:internal/test_runner/test:806:20\n        at async Suite.runHook (node:internal/test_runner/test:804:7)\n        at async Test.run (node:internal/test_runner/test:863:9) {\n      errno: 1,\n      code: 'EPERM',\n      path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n      syscall: 'rm'\n    }\n\n  ✖ should run new tests when a new file is created in the watched directory (0.5452ms)\n    Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n        at Object.rmSync (node:fs:1246:18)\n        at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n        at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n        at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n        at TestHook.runInAsyncScope (node:async_hooks:211:14)\n        at TestHook.run (node:internal/test_runner/test:887:25)\n        at TestHook.run (node:internal/test_runner/test:1157:18)\n        at node:internal/test_runner/test:806:20\n        at async Suite.runHook (node:internal/test_runner/test:804:7)\n        at async Test.run (node:internal/test_runner/test:863:9) {\n      errno: 1,\n      code: 'EPERM',\n      path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n      syscall: 'rm'\n    }\n\n▶ test runner watch mode (24797.1429ms)\nℹ tests 7\nℹ suites 1\nℹ pass 2\nℹ fail 5\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 25037.4827\n\n✖ failing tests:\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:169:3\n✖ should run tests repeatedly (6387.6265ms)\n  AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n  \n  '✔ test has ran (18.1095ms)\\n' +\n    '✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (1058.8466ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✔ test has ran (23.9442ms)\\n' +\n    '✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (957.6018ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✔ test has ran (17.7252ms)\\n' +\n    'ℹ tests 5\\n' +\n    'ℹ suites 0\\n' +\n    'ℹ pass 3\\n' +\n    'ℹ fail 2\\n' +\n    'ℹ cancelled 0\\n' +\n    'ℹ skipped 0\\n' +\n    'ℹ todo 0\\n' +\n    'ℹ duration_ms 1181.2087\\n'\n  \n      at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:80:14)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:161:26)\n      at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:170:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 0)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: true,\n    code: 'ERR_ASSERTION',\n    actual: \"✔ test has ran (18.1095ms)\\n✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (1058.8466ms)\\n  'test failed'\\n\\n✔ test has ran (23.9442ms)\\n✖ c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js (957.6018ms)\\n  'test failed'\\n\\n✔ test has ran (17.7252ms)\\nℹ tests 5\\n...\",\n    expected: /tests 1/,\n    operator: 'match'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:181:3\n✖ should support running tests without a file (6053.9114ms)\n  AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n  \n  '✖ test.js (1031.0906ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✔ test has ran (10.3761ms)\\n' +\n    '✖ test.js (1016.3231ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✔ test has ran (27.7831ms)\\n' +\n    'ℹ tests 4\\n' +\n    'ℹ suites 0\\n' +\n    'ℹ pass 2\\n' +\n    'ℹ fail 2\\n' +\n    'ℹ cancelled 0\\n' +\n    'ℹ skipped 0\\n' +\n    'ℹ todo 0\\n' +\n    'ℹ duration_ms 1112.9031\\n'\n  \n      at testUpdate (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:80:14)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async testWatch (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:161:26)\n      at async TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:182:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:584:7) {\n    generatedMessage: true,\n    code: 'ERR_ASSERTION',\n    actual: \"✖ test.js (1031.0906ms)\\n  'test failed'\\n\\n✔ test has ran (10.3761ms)\\n✖ test.js (1016.3231ms)\\n  'test failed'\\n\\n✔ test has ran (27.7831ms)\\nℹ tests 4\\nℹ suites 0\\n...\",\n    expected: /tests 1/,\n    operator: 'match'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:185:3\n✖ should support a watched test file rename (4010.2208ms)\n  Error: ENOENT: no such file or directory, rename 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test.js' -> 'c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687\\test-renamed-test.js'\n      at renameSync (node:fs:1020:11)\n      at Timeout._onTimeout (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:93:40)\n      at listOnTimeout (node:internal/timers:614:17)\n      at process.processTimers (node:internal/timers:549:7) {\n    errno: -4058,\n    code: 'ENOENT',\n    syscall: 'rename',\n    path: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test.js',\n    dest: 'c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687\\\\test-renamed-test.js'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:189:3\n✖ should not throw when delete a watched test file (8.1859ms)\n  Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n      at Object.rmSync (node:fs:1246:18)\n      at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n      at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n      at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n      at TestHook.runInAsyncScope (node:async_hooks:211:14)\n      at TestHook.run (node:internal/test_runner/test:887:25)\n      at TestHook.run (node:internal/test_runner/test:1157:18)\n      at node:internal/test_runner/test:806:20\n      at async Suite.runHook (node:internal/test_runner/test:804:7)\n      at async Test.run (node:internal/test_runner/test:863:9) {\n    errno: 1,\n    code: 'EPERM',\n    path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n    syscall: 'rm'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:193:3\n✖ should run new tests when a new file is created in the watched directory (0.5452ms)\n  Error: EPERM, Permission denied: \\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687 '\\\\?\\c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\.tmp.687'\n      at Object.rmSync (node:fs:1246:18)\n      at rmSync (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:20:8)\n      at Object.refresh (c:\\workspace\\node-test-binary-windows-js-suites\\node\\test\\common\\tmpdir.js:35:3)\n      at TestContext.refresh (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:33:10)\n      at TestHook.runInAsyncScope (node:async_hooks:211:14)\n      at TestHook.run (node:internal/test_runner/test:887:25)\n      at TestHook.run (node:internal/test_runner/test:1157:18)\n      at node:internal/test_runner/test:806:20\n      at async Suite.runHook (node:internal/test_runner/test:804:7)\n      at async Test.run (node:internal/test_runner/test:863:9) {\n    errno: 1,\n    code: 'EPERM',\n    path: '\\\\\\\\?\\\\c:\\\\workspace\\\\node-test-binary-windows-js-suites\\\\node\\\\test\\\\.tmp.687',\n    syscall: 'rm'\n  }"},"time":30325.219,"fullname":"parallel/test-runner-watch-mode","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725262864654,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #29840","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/29840/","buildNumber":29840,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"23.0.0","commitHash":"09dd58319e84d26d6053f1ba5a4a739257c767fa","callStack":[{"upstreamBuild":65401,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73805,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61811,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54552"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":113,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4436.4082ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4897.3056ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4997.7015ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (4998.9144ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (4663.7111ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4817.2536ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4427.759ms)\n  ✖ async-hooks\\test-async-local-storage-gcable.js (4550.7794ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 7)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (5107.2436ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (4208.5513ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4249.8534ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (4316.396ms)\n  ✖ async-hooks\\test-async-local-storage-no-mix-contexts.js (5325.0964ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-no-mix-contexts.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 12)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-promises.js (5558.2064ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (4958.9442ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (5278.0788ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5262.8487ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4284.7956ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4304.2618ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (5433.1861ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4958.3308ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (4972.8555ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (4644.2512ms)\n▶ AsyncContextFrame (6258.3973ms)\nℹ tests 23\nℹ suites 1\nℹ pass 21\nℹ fail 2\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 6275.588\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-gcable.js (4550.7794ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-gcable.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 7)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-no-mix-contexts.js (5325.0964ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-no-mix-contexts.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 12)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":6745.919,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}},{"testName":"sequential/test-http2-large-file","timestamp":1725262864654,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win2019-COMPILED_BY-vs2022 #29840","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2019-COMPILED_BY-vs2022/29840/","buildNumber":29840,"builtOn":"test-rackspace-win2019_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"09dd58319e84d26d6053f1ba5a4a739257c767fa","callStack":[{"upstreamBuild":65401,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73805,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61811,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54552"}],"tap":{"ok":false,"name":"sequential/test-http2-large-file","id":990,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout"},"time":120165.974,"fullname":"sequential/test-http2-large-file","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725262864654,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #29840","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/29840/","buildNumber":29840,"builtOn":"test-rackspace-win2022_vs2022-x64-6","nodeVersion":"23.0.0","commitHash":"09dd58319e84d26d6053f1ba5a4a739257c767fa","callStack":[{"upstreamBuild":65401,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73805,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61811,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54552"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":110,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4468.8944ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4944.9404ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4682.4376ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5213.4286ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (5052.0611ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (5283.7983ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4608.4388ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (5130.6982ms)\n  ✖ async-hooks\\test-async-local-storage-http-agent.js (5096.0183ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-http-agent.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 8)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-http.js (5130.9123ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4945.1613ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (4758.963ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5232.2019ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5093.521ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (4866.3201ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (4268.1176ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5165.1324ms)\n  ✔ parallel\\test-async-local-storage-bind.js (5080.0612ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4960.0268ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4463.9206ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4830.2013ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5231.181ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (4947.9121ms)\n▶ AsyncContextFrame (5734.4666ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 5749.4725\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-http-agent.js (5096.0183ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-http-agent.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 8)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":6280.31,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}